index.html

```diff
@@ -321,7 +340,7 @@ <h3>legend</h3>
 
           // Open the row that is now the target.
           var row = ptable.row( tr );
```

Comment:
This could call `open_row(tr)`, right?  There is a little more indirection, but you aren't hitting the "private" API.  And if you do that, you no longer need a separate "private" API, you can just roll _open_row into open_row and _close_row into close_row.

Reply:
Ah, true.  I was thinking of `open_row` as a function that took the row index as an argument, but really it takes whatever [`ptable.row()`](https://datatables.net/reference/api/row()) takes as an argument, i.e., a [row selector](https://datatables.net/reference/type/row-selector).  So I can pass the row index in all the other cases and the `<tr>` node in this case.

Reply:
Fixed in the updated version (I force-pushed since I updated all three patches in place, so I can still cleanly push the queue of 3 patches).